AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -8,6 +8,10 @@
 ! @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js
 @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=nintendo.*|mtv.com|ryobitools.eu|cc.com|3sat.de|zdf.de|costco.com|giantbomb.com|glovoapp.com|rte.ie|digitaltrends.com|spiele.bild.de|spiele.stern.de|vanityfair.com|saarbruecker-zeitung.de|olympics.com|dziendobry.tvn.pl|bergfreunde.de|oe24.at|epicgames.com|toteraz.pl|natgeotv.com|midomi.com|volksfreund.de|kawasaki.com|foxchannels.com.tr|ga.de|shop.mango.com|kralmuzik.com.tr|westernunion.com|roastmarket.de|new.cappex.com|vimeo.com|heute.at|ntv.com.tr|wargaming.net|metacritic.com|gq-magazine.co.uk|viafree.dk|viafree.fi|viafree.no|viafree.se|glamour.de|cbsnews.com|worldoftanks.ru|vg247.com|newyorker.com|games.miamiherald.com|eurogamer.es|eurogamer.de|bet.com|ikonpass.com|mcdelivery.de|id.trimble.com|skateboarding.transworld.net|newegg.com|docs.vmware.com|udemy.com|olx.pl|usmagazine.com|uefa.com|milwaukeetool.eu|muscleandfitness.com|20min.ch|ted.com|elisa.fi|nhl.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/135197
+!#if (!adguard_app_android && !adguard_app_mac && !adguard_app_windows && !adguard_ext_chromium && !adguard_ext_edge && !adguard_ext_firefox && !adguard_ext_safari )
```

Comment:
We have these platforms:
```
adguard_app_windows      -- AG for Windows
adguard_app_mac             -- AG for Mac
adguard_app_android        -- AG for Android
adguard_app_ios                -- AG for iOS
adguard_ext_chromium      -- AG browser extension for Chrome
adguard_ext_firefox          -- AG browser extension for Firefox
adguard_ext_edge             -- AG browser extension for Edge
adguard_ext_safari            -- AG browser extension for Safari
adguard_ext_opera            -- AG browser extension for Opera
adguard_ext_android_cb    -- AG content blocker for Samsung/Yandex
ext_ublock                           -- uBlock
```
Where this rule must be applied? And why it must be applied to not all platforms?

Reply:
Hmm... I forgot to add `adguard_ext_opera`.
I think that the above rule is needed on partial AdGuard platforms that cannot apply a JS rule.